docs/api/_static/css/inrupt.css

```diff
@@ -204,6 +204,11 @@ p.topic-title {
   color: #222222;
 }
 
+.social-icon {
+  padding-left: 0.12rem;
```

Comment:
These icons are currently pretty hard to hit for users with motor impairments or on touch screens, so maybe we could have some extra padding here?